versions/1/agent/agent.spec.yml

```diff
@@ -0,0 +1,7 @@
+##
+## Describes the specification for an input template file
+##
+spec:
+  # Everything under here follows JSON schema (https://json-schema.org/), written as YAML for readability
+  type: object
+  additionalProperties: true
```

Comment:
I understand that this schema file describes the `*.yml.hbs` file, which is Handlebars template. I think it's not possible to easily define a JSON schema for this file as the JSON format can be broken by template placeholders.